src/features/P0_create_package_catalog.py

```diff
@@ -1,20 +1,18 @@
 import os
 import subprocess
-import csv
 import copy
 from datetime import datetime
-from lib.logging import setup_log_level
-from lib.parser import setup_command_line
 
 
-def create_csv(data, logging):
+def create_csv(data, date_now, logging, lts):
     logging.info("Writing results to file")
     try:
-        date_now = datetime.now().strftime("%d-%m-%Y-%H-%M-%S")
-        csv_file_name = "packages-catalog-{date}.csv".format(date=date_now)
+        csv_file_name = "C:/Users/nicol/Documents/GitHub/stackage-evolution/data/test/%s/packages-catalog-{date}.csv".format(
+            date=date_now) % lts

```

Comment:
Trata de usar path relativos y reutizables en cualquier ambiente

```suggestion
        csv_file_name = os.path.join(os.path.dirname(__file__), f'../../data/test/{lts}/packages-catalog-{date}.csv')
```